packages/web-pkg/src/components/FilesList/ResourceTile.vue

```diff
@@ -178,11 +178,11 @@ export default defineComponent({
       statusIconAttrs,
       showStatusIcon,
       tooltipLabelIcon,
-      resourceDisabled,
       resourceDescription,
       shouldDisplayThumbnails
     }
-  }
+  },
+  methods: { isProjectSpaceResource }
```

Comment:
Can you try returning this method via `setup`? Or have a wrapper instead of using the options API.

Reply:
my gosh sometimes my IDE is just doing that since the update -.-